README.md

```diff
@@ -24,12 +24,13 @@ additional metrics!
 
 ### WARNING: THESE METRIC/TAG NAMES ARE UNSTABLE AND MAY CHANGE IN A FUTURE RELEASE.
 
-| Metric name | Labels/tags |
-| ------------- | ------------- |
-| nodes | ready=&lt;true\|false&gt; |
-| deployment_replicas | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
-| deployment_replicas_available | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
-| container_restarts | name=&lt;container-name&gt; namespace=&lt;pod-namespace&gt; pod_name=&lt;pod-name&gt; |
+| Metric name| Metric type | Labels/tags |
+| ---------- | ----------- | ----------- |
+| node_info | Gauge | node=&lt;node-address&gt; kernel_version=&lt;kernel-version&gt; os_image=&lt;os-image-name&gt; container_runtime_version=&lt;container-runtime-and-version-combination&gt; kubelet_version=&lt;kubelet-version&gt; kubeproxy_version=&lt;kubeproxy-version&gt; |
+| node_status_ready| Gauge | node=&lt;node-address&gt; condition=&lt;true\|false\|unknown&gt; |
+| deployment_replicas | Gauge | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
+| deployment_replicas_available | Gauge | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
+| pod_container_restarts | Counter | container=&lt;container-name&gt; namespace=&lt;pod-namespace&gt; pod=&lt;pod-name&gt; |
```

Comment:
Any particular reason for switching to escaped `<` and `>` or is your editor doing that?
This is fine for now. As we grow the metrics, we should probably break this out by collector.

One line per label will make it more readable for a lot of cases.